.github/workflows/ci.yml

```diff
@@ -34,13 +34,13 @@ jobs:
     # Run tests that require headfull Chrome.
     - run: sudo apt-get install xvfb
     - name: yarn test-clients
-      run: xvfb-run --auto-servernum yarn test-clients
+      run: xvfb-run --auto-servernum bash $GITHUB_WORKSPACE/.github/scripts/git-get-shared-history.sh yarn test-clients
     - name: yarn test-docs
       run: xvfb-run --auto-servernum yarn test-docs
     - name: yarn test-viewer
-      run: xvfb-run --auto-servernum yarn test-viewer
+      run: xvfb-run --auto-servernum bash $GITHUB_WORKSPACE/.github/scripts/git-get-shared-history.sh yarn test-viewer
     - name: yarn test-treemap
-      run: xvfb-run --auto-servernum yarn test-treemap
+      run: xvfb-run --auto-servernum bash $GITHUB_WORKSPACE/.github/scripts/git-get-shared-history.sh yarn test-treemap
```

Comment:
Should these be using `jest-retry.sh` instead?

Reply:
lol